packages/keyring/src/pair/index.ts

```diff
@@ -115,9 +115,7 @@ export function createPair ({ toSS58, type }: Setup, { publicKey, secretKey }: P
   const encodeAddress = (): string => {
     const raw = TYPE_ADDRESS[type](publicKey);
 
-    return type === 'ethereum'
-      ? ethereumEncode(raw)
-      : toSS58(raw);
+    return type === 'ethereum' ? (ethereumEncode(raw)) : toSS58(raw);
```

Comment:
```suggestion
    return type === 'ethereum'
      ? ethereumEncode(raw)
      : toSS58(raw);
```

The original here is fine, not 100% sure why the ()'s slipped in.